ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-2023a.yaml

```diff
@@ -0,0 +1,407 @@
+  to: jupyter-minimal-ubi8-python-3.8
+- context_dir: jupyter/datascience/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: jupyter-minimal-ubi8-python-3.8
+  to: jupyter-datascience-ubi8-python-3.8
+- context_dir: jupyter/pytorch/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: jupyter-datascience-ubi8-python-3.8
+  to: jupyter-pytorch-ubi8-python-3.8
+- context_dir: cuda/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: base-ubi8-python-3.8
+  to: cuda-ubi8-python-3.8
+- context_dir: jupyter/minimal/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: cuda-ubi8-python-3.8
+  to: cuda-jupyter-minimal-ubi8-python-3.8
+- context_dir: jupyter/datascience/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: cuda-jupyter-minimal-ubi8-python-3.8
+  to: cuda-jupyter-datascience-ubi8-python-3.8
+- context_dir: jupyter/tensorflow/ubi8-python-3.8
+  dockerfile_path: Dockerfile
+  from: cuda-jupyter-datascience-ubi8-python-3.8
+  to: cuda-jupyter-tensorflow-ubi8-python-3.8
+- context_dir: base/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  to: base-ubi9-python-3.9
+- context_dir: jupyter/minimal/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.9
+  to: jupyter-minimal-ubi9-python-3.9
+- context_dir: jupyter/datascience/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: jupyter-minimal-ubi9-python-3.9
+  to: jupyter-datascience-ubi9-python-3.9
+- context_dir: jupyter/pytorch/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: jupyter-datascience-ubi9-python-3.9
+  to: jupyter-pytorch-ubi9-python-3.9
+- context_dir: cuda/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.9
+  to: cuda-ubi9-python-3.9
+- context_dir: jupyter/minimal/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: cuda-ubi9-python-3.9
+  to: cuda-jupyter-minimal-ubi9-python-3.9
+- context_dir: jupyter/datascience/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: cuda-jupyter-minimal-ubi9-python-3.9
+  to: cuda-jupyter-datascience-ubi9-python-3.9
+- context_dir: jupyter/tensorflow/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: cuda-jupyter-datascience-ubi9-python-3.9
+  to: cuda-jupyter-tensorflow-ubi9-python-3.9
+promotion:
+  namespace: opendatahub-io
+  tag: weekly
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 400Mi
+tests:
+- as: notebook-base-ubi8-python-3-8-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: base-ubi8-python-3.8
+    env:
+      IMAGE_REPO: workbench-images
+      IMAGE_TAG: YearIndex
+      RELEASE_VERSION: base-ubi8-python-3.8-2023a
+    workflow: opendatahub-io-ci-image-mirror
+- as: notebook-base-ubi8-python-3-8-image-mirror-weekly
+  cron: 0 12 * * 2
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: base-ubi8-python-3.8
+    env:
+      IMAGE_REPO: workbench-images
+      IMAGE_TAG: weekly
```

Comment:
Note : With the current logic of the workflow, adding just a `weekly` tag will create a mirror image as follows:
`workbench-images:weekly`.

I think we would want to  append the `RELEASE-VERSION` as well.

Reply:
To add release version, we can update the workflow as follows-
```
if [[ "$IMAGE_TAG" == "weekly" ]]; then
    case "$JOB_TYPE" in
        periodic)
            log "INFO Building weekly image tag for a $JOB_TYPE job"
            if [[ -n "${RELEASE_VERSION-}" ]]; then
                IMAGE_TAG="${RELEASE_VERSION}-${IMAGE_TAG}"
            fi
            ;;
        *)
            IMAGE_TAG=${IMAGE_TAG}
            ;;
    esac
fi
```